devicetypes/smartthings/zwave-motion-sensor.src/zwave-motion-sensor.groovy

```diff
@@ -142,18 +142,40 @@ def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cm
 	result
 }
 
+def zwaveEvent(physicalgraph.zwave.commands.associationv2.AssociationReport cmd) {
+	def result = []
+
+	if (cmd.groupingIdentifier == 1) {
+		if (cmd.nodeId.any { it == zwaveHubNodeId }) {
+			state.group1Assoc = true
+			result << createEvent(descriptionText: "$device.displayName is associated in group ${cmd.groupingIdentifier}", isStateChange: false)
```

Comment:
should these be events or debug logs?

Reply:
I ended up going with events, like in the Wake Up Notification (`createEvent(descriptionText: "${device.displayName} woke up", isStateChange: false)`), so that they might be more readily accessible in the event list for troubleshooting a customer issue. Though, log.info could be equally as useful. Either shouldn't hurt the system (though, given some changes, it is possible that those events may stop working).